source/tools/cdk-integ-tools/lib/canonicalize-assets.ts

```diff
@@ -39,6 +41,24 @@
     ]);
   }
 
+  function checkIgnoreList(functionName: string): boolean {
```

Comment:
Don't hide a function inside another function (that's the case here, right? Hard to tell in github).